### PR TITLE
Cleanup fork

### PR DESCRIPTION
**Pull request by ffromani**
Cleanup our fork enforcing clearer separation between assets from u/s - which we inherited and we will update as-is, and kni-specific assets.

### CODE REVIEW

**Comment by ffromani** [comment]
CI will fail because of the Makefile rename. prow PR: https://github.com/openshift/release/pull/23614

**Comment by ffromani** [comment]
> @fromanirh: The following tests **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> Test name 	Commit 	Details 	Required 	Rerun command
> ci/prow/ci-unit 	[40fedc5](https://github.com/openshift-kni/scheduler-plugins/commit/40fedc5511cbd7675f8f688e8bd98bc7ffc7be04) 	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-kni_scheduler-plugins/11/pull-ci-openshift-kni-scheduler-plugins-master-ci-unit/1460181598744875008) 	true 	`/test ci-unit`
> ci/prow/images 	[40fedc5](https://github.com/openshift-kni/scheduler-plugins/commit/40fedc5511cbd7675f8f688e8bd98bc7ffc7be04) 	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-kni_scheduler-plugins/11/pull-ci-openshift-kni-scheduler-plugins-master-images/1460181598791012352) 	true 	`/test images`
> 
> [Full PR test history](https://prow.ci.openshift.org/pr-history?org=openshift-kni&repo=scheduler-plugins&pr=11). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:fromanirh).

Expected. Fix in: openshift/release#23614

File: Makefile.kni [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 The Kubernetes Authors.
```
Comment (yanirq) [inline Makefile.kni:1]: this will need to be adjusted in openshift/release

Comment (ffromani) [reply Makefile.kni:1]: sure: https://github.com/openshift/release/pull/23614

**Pull request merged**